src/main/java/org/helpberkeley/memberdata/WorkflowExporter.java

```diff
@@ -22,58 +22,63 @@
  */
 package org.helpberkeley.memberdata;
 
+import java.text.MessageFormat;
 import java.util.*;
 
 public class WorkflowExporter extends Exporter {
 
-    WorkflowBean bean;
-    private StringBuilder updateWarnings;
-    private WorkflowParser parser;
-    private List<WorkflowBean> updatedBeans;
+    public static final String TOO_MANY_MEMBERS_ERROR = "This is many more members (at least {0}) than the typical run, " +
+            "does this sheet contain more members than just the drivers and consumers for this run?\n";
+    public static final String NO_MATCHING_MEMBER_ERROR = "UserName {0} at line {1}" +
+            " does not match any current members, please update to a current member.\n";
+    public static final String DRIVER_IS_CONSUMER_ERROR = "Line number {0} begins with TRUE TRUE. " +
+            "Is this a driver who is also a consumer? If so, the consumer column must be set to false.\n";
+
+    private final StringBuilder updateWarnings = new StringBuilder();
+    private final WorkflowParser parser;
+    private final List<WorkflowBean> updatedBeans = new ArrayList<>();
     private int numMembers;
```

Comment:
This doesn't need to be a class member.  Make it a local variable in updateMemberData.